docker/internal-auth/env.template

```diff
@@ -0,0 +1,65 @@
+# This should be the URL your end-users will connect to
+BASE_URL=https://example.fiftyone.ai
+
+# This should be set to the URI your end-users will use to connect to the API
+#  This could be the same as AUTH0_BASE_URL
+FIFTYONE_API_URI=https://example-api.fiftyone.ai
+
+# This should be a MongoDB Connection String for your database
+FIFTYONE_DATABASE_URI="mongodb://username:password@mongodb-example.fiftyone.ai:27017/?authSource=admin"
+# If you are using a different MongoDB Connection String for your CAS database,
+#  set it here
+# CAS_DATABASE_URI=""mongodb://username:password@mongodb-cas-example.fiftyone.ai:27017/?authSource=admin""
+
+# FIFTYONE_AUTH_SECRET is a random string used to authenticate to the CAS service
+FIFTYONE_AUTH_SECRET=
```

Comment:
😕 Should the user generate a random string? If so, how long? In other cases, we provide suggestions for generating using `openssl` or python's `cryptography` library.

What does it mean to be use to authenticate the CAS service?

Reply:
let me know if the additional documentation is sufficient to explain it

Reply:
It does!